src/app/App/AppContainer.js

```diff
@@ -3,10 +3,14 @@ import { withRouter } from 'react-router'
 import { connect } from 'react-redux'
 import { App } from './App'
 import { maintenanceSelector } from '../../redux/selectors/maintenanceSelector'
+import { selectCurrentUser } from '../../redux/selectors/currentUserSelector'
 
 export const mapStateToProps = state => {
+  const user = selectCurrentUser(state)
+  const isUserConnected = user != null
```

Comment:
```suggestion
  const isUserConnected = selectCurrentUser(state) != null
```
Le nom de la fonction me semble assez explicite pour éviter d'avoir une variable, qu'en penses tu ?